solr/core/src/java/org/apache/solr/handler/IndexFetcher.java

```diff
@@ -1831,21 +1814,27 @@ private void fetch() throws Exception {
     private int fetchPackets(FastInputStream fis) throws Exception {
       byte[] intbytes = new byte[4];
       byte[] longbytes = new byte[8];
+      boolean isContentReceived = false;
```

Comment:
We probably don't need this variable anymore; can rely on bytesDownloaded > 0 condition.